src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
It appears that there is a duplicate getRandomColor method, one of which is static and the other is not. You should have only one method, and it can be made static.